services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -669,13 +723,21 @@ public void depart_at() throws Exception {
   @Test
   public void enable_refresh() throws Exception {
     MapboxDirections mapboxDirections = MapboxDirections.builder()
+      .accessToken("token")
       .routeOptions(routeOptions.toBuilder().baseUrl(mockUrl.toString()).build())
       .build();
 
     assertEquals("true",
       mapboxDirections.cloneCall().request().url().queryParameter("enable_refresh"));
   }
 
+  @Test(expected = Exception.class)
+  public void token_required() throws Exception {
```

Comment:
do we provide any readable message?
can we test it?

Reply:
We can't this is implemented directly in the `AutoValue` library - if there's a non-nullable field that wasn't initialized, `AutoValue` will throw an exception saying which field is missing. I added the test mostly for us to remember to keep the `accessToken` as a `non-nullable field in the future.